timelock-server/build.gradle

```diff
@@ -39,11 +45,44 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-core'
     testCompile project(":atlasdb-tests-shared")
 
-    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
 }
 
 distribution {
     serviceName "timelock-server"
     mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
     args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-Xms512m",
```

Comment:
nit: I'm worried this could have unintended consequences that could be conflated with the http2 change we are attempting to examine.  Wondering if we should delay this heap setting to a separate PR.